app/views/admin/response_templates/_form.html.erb

```diff
@@ -1,25 +1,32 @@
 <%= form_with(model: [:admin, response_template], local: true, html: { class: "crayons-card p-6" }) do |f| %>
-  <div class="form-group">
-    <%= f.label :type_of, "Type of Response Template" %>
-    <% type_of_options = [["Trusted User Comment", "mod_comment"], ["Personal Comment", "personal_comment"], %w[Email email_reply], ["Abuse Report Email", "abuse_report_email_reply"]] %>
-    <%= f.select :type_of, options_for_select(type_of_options, response_template.type_of), html: { required: true }, class: "form-control" %>
+  <div class="flex flex-col gap-4">
+    <div class="crayons-field">
+      <%= label_tag :type_of, "Type of Response Template", class: "crayons-field__label" %>
+      <% type_of_options = [["Trusted User Comment", "mod_comment"], ["Personal Comment", "personal_comment"], %w[Email email_reply], ["Abuse Report Email", "abuse_report_email_reply"]] %>
+      <%= select_tag :type_of, options_for_select(type_of_options, response_template.type_of), html: { required: true }, class: "crayons-select" %>
+    </div>
```

Comment:
The change from `f.label` to `label_tag` and `f.select` to `select_tag` changes the way the form params are being passed on this form. Is there a reason to prefer this style over the previous one?

Some more context on this: When using the `f.select` the parameter arrives embedded inside `params[: response_template][:type_of]`. With this change the params arrive directly as `params[:type_of]` and I see this error when trying to create a new response template

<img width="1107" alt="Screen Shot 2022-12-19 at 22 26 48" src="https://user-images.githubusercontent.com/6045239/208584406-89107b26-d8f9-4afd-b0d7-1d78dfaf4dc4.png">

One suggestion to make this work would be to replace the tags with something like this (for all of the tags in this form):

```suggestion
      <%= label_tag response_template[type_of], "Type of Response Template", class: "crayons-field__label" %>
      <% type_of_options = [["Trusted User Comment", "mod_comment"], ["Personal Comment", "personal_comment"], %w[Email email_reply], ["Abuse Report Email", "abuse_report_email_reply"]] %>
      <%= select_tag response_template[type_of], options_for_select(type_of_options, response_template.type_of), html: { required: true }, class: "crayons-select" %>
    </div>
```

This way the parameter will arrive to the Rails controller the way it's expected (inside a `response_template` hash). There's many ways to implement this in Rails and I don't have a strong opinion on which way to do it here, this was just my suggestion to keep the form working. Both a pro and a con that helpers can be used in many ways to achieve the same result 😅

Reply:
Thanks for the detailed explanation. This change was applied when I was comparing it with other files and saw that somewhere it was getting used like the code that I mentioned and also it was working locally so thought that this is correct.

Thanks for explaining this. I have applied your suggestions to all tags.